src/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import MaterialAvatar from '@material-ui/core/Avatar'
+import withStyles from '@material-ui/core/styles/withStyles'
+import defaultImage from './default.jpg'
+
+const DEFAULT_AVATAR_SIZE = 68
+
+const styles = theme => ({
+  root: {
+    backgroundColor: theme.palette.grey[300],
+    border: `solid 1px ${theme.palette.grey[300]}`
+  }
+})
+
+/**
+ * A circle avatar with a light border.
+ *
+ * ```js
+ * import { Avatar } from '@theconversation/ui'
+ *
+ * <Avatar src={YOUR_URL} />
+ * ```
+ */
+export const Avatar = ({ src, size, classes }) => {
+  return (
+    <MaterialAvatar src={src} classes={classes} style={{ width: size, height: size }} />
+  )
+}
```

Comment:
Is it worth it adding `...other` to the `Avatar` props, so any other MUI can be passed down to the [`MaterialAvatar` API](https://material-ui.com/api/avatar/)?